tokio-console/src/view/resources.rs

```diff
@@ -59,15 +71,15 @@ impl TableList for ResourcesTable {
             .sort_by
             .sort(now, &mut table_list_state.sorted_items);
 
-        let viz_len: u16 = Self::HEADER[6].len() as u16;
+        let viz_len: u16 = Self::WIDTHS[6] as u16;
 
-        let mut id_width = view::Width::new(Self::HEADER[0].len() as u16);
-        let mut parent_width = view::Width::new(Self::HEADER[1].len() as u16);
+        let mut id_width = view::Width::new(Self::WIDTHS[0] as u16);
+        let mut parent_width = view::Width::new(Self::WIDTHS[1] as u16);
 
-        let mut kind_width = view::Width::new(Self::HEADER[2].len() as u16);
-        let mut target_width = view::Width::new(Self::HEADER[4].len() as u16);
-        let mut type_width = view::Width::new(Self::HEADER[5].len() as u16);
-        let mut location_width = view::Width::new(Self::HEADER[7].len() as u16);
+        let mut kind_width = view::Width::new(Self::WIDTHS[2] as u16);
+        let mut target_width = view::Width::new(Self::WIDTHS[4] as u16);
+        let mut type_width = view::Width::new(Self::WIDTHS[5] as u16);
+        let mut location_width = view::Width::new(Self::WIDTHS[7] as u16);
```

Comment:
The `WIDTHS` array seems fine to me, but, for what it's worth, think it would *also* be fine to just change this code to something like
```rust
        let viz_len: u16 = (Self::HEADER[6].len() + 1) as u16;

        let mut id_width = view::Width::new((Self::HEADER[0].len() + 1) as u16);
        let mut parent_width = view::Width::new((Self::HEADER[1].len() + 1) as u16);
        // ... and so on ...
```

Since the `HEADER` array is a const, and the `1` is also a constant, this calculation should probably get const-folded, so I don't think there would be a performance difference between this approach and creating a separate const array. I think the code might be a bit simpler if we avoided the separate array.

But, this is fine either way --- it's not a big deal.